README.adoc

```diff
@@ -95,7 +95,7 @@ in the `<dependencyManagement/>` section of the `pom.xml` file.
 For the Library project, you need not add dependencies. The basic `spring-boot-starter`
 dependency provides everything you need.
 
-You can get a Maven build file with the necessary dependencies directly from the https://start.spring.io/#!type=maven-project&language=java&platformVersion=2.4.3.RELEASE&packaging=jar&jvmVersion=1.8&groupId=com.example&artifactId=multi-module-library&name=multi-module-library&description=Demo%20project%20for%20Spring%20Boot&packageName=com.example.multi-module-library[Spring Initializr].
+You can get a Maven build file with the necessary dependencies directly from the https://start.spring.io/#!type=maven-project&language=java&platformVersion=3.1.5&packaging=jar&jvmVersion=17&groupId=com.example&artifactId=multi-module-library&name=multi-module-library&description=Demo%20project%20for%20Spring%20Boot&packageName=com.example.multi-module-library[Spring Initializr].
```

Comment:
In the readme files, our current approach is to remove the `platformVersion` and `jvmVersion` (and their preceding ampersands and their values) rather than specify them. That saves us the trouble of updating the readme for each version. Please make that change instead. Your intention was good and is appreciated, but you had no way to know that we manage this problem differently.

Reply:
That's a great idea. Indeed, I did not know these params could be omitted.

Reply:
The Initializr has defaults for everything but the actual projects that (Spring Web, Spring Security, and so on) that someone attaches. A useful feature.